### PR TITLE
Colour current graphviz nodes

### PR DESCRIPTION
**Pull request by Svenito**
Colours current node in Graphviz output (`ag` command) 'lightblue'.

Add `graph.gv.current` option to config with default to `false`

### CODE REVIEW

**Comment by radare** [comment]
Can you make the `"yellow":"lightgray"` configurable? using the RCons palette? this will make graphviz, ascii art and html graphs color-consistent

**Comment by Svenito** [comment]
Yup, I'll get on that too.

**Comment by radare** [comment]
cool! thanks

**Comment by Svenito** [comment]
Should I use the colours to match the ASCII graph, or is it ok to define some new ones that work with the GV output? `blue` isn't very readable in GV nodes for instance. I can probably match the other colours, and use `gray` instead of `lightgrey`.

**Comment by radare** [comment]
I think its ok to have graphviz-specific setup

**Comment by Svenito** [comment]
Ok, shall I make a whole new palette entry for GV in that case?

**Comment by radare** [comment]
Add those variables under graph.gv. as eval vars

**Comment by radare** [comment]
better not to mess with RCons for this

**Comment by Svenito** [comment]
Using the existing colours, as I wasn't sure if it was right to add new term colour escape codes. Tried to match existing way colours are set.

If these colours are not ok, I'll happily change them.

File: libr/core/anal.c [hunk 1]
```diff
@@ -1242,7 +1246,7 @@ R_API void r_core_anal_coderefs(RCore *core, ut64 addr, int fmt) {
 					if (!gv_edge || !*gv_edge)
 						gv_edge = "arrowhead=\"vee\"";
 					if (!gv_node || !*gv_node)
-						gv_node = "color=lightgray, style=filled shape=box";
+						gv_node = "color=gray, style=filled shape=box";
```
Comment (radare) [inline libr/core/anal.c]: isnt gray configurable?

Comment (Svenito) [inline libr/core/anal.c]: missed it... yes it is

**Comment by radare** [comment]
Thanks!

**Pull request merged**